src/unix/linux_like/linux/musl/b32/arm/mod.rs

```diff
@@ -39,12 +59,27 @@ s! {
         pub st_size: off_t,
         pub st_blksize: crate::blksize_t,
         pub st_blocks: crate::blkcnt_t,
+        #[cfg(musl32_time64)]
+        __st_atim32: __c_anonymous_timespec32,
+        #[cfg(musl32_time64)]
+        __st_mtim32: __c_anonymous_timespec32,
+        #[cfg(musl32_time64)]
+        __st_ctim32: __c_anonymous_timespec32,
+        #[cfg(musl32_time64)]
+        pub st_ino: crate::ino_t,
```

Comment:
I can't comment on the right lines, but isn't `struct stat64` identical here? If so, I think you could delete the struct and turn it into a type alias.

Reply:
Yeah - I checked the musl source code and I see that `stat64` is defined as `stat` - I'll double check each arch in musl and merge as necessary

Reply:
Yeah - I checked the musl source code and I see that `stat64` is defined as `stat` - I'll double check each arch in musl and merge as necessary. I noticed that __pad1 and __pad2 are pub in loongarch64, I'm not sure if making these private is acceptably breaking or not.